app/test_main.py

```diff
@@ -1,3 +1,79 @@
+import pytest
+from typing import Any
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_year,dog_year,expected_result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2])
+    ]
+)
+def test_should_equal_expected_result(
+    cat_year: int,
+    dog_year: int,
+    expected_result: list[int]
+) -> None:
+    assert get_human_age(cat_year, dog_year) == expected_result
+
+
+@pytest.mark.parametrize(
+    "cat_year,dog_year",
+    [
+        (15, 15),
+        (23, 23),
+        (17, 17)
+    ]
+)
+def test_should_not_change_previous_value_if_output_change_some_integer(
```

Comment:
The test name `test_should_not_change_previous_value_if_output_change_some_integer` is unclear. Consider renaming it to something more descriptive of the test's purpose, such as `test_should_return_one_for_specific_years`.